lib/cookie.js

```diff
@@ -1,11 +1,23 @@
 const request = require('request')
 const _ = require('lodash')
 
-module.exports = function (tokens, url) {
+exports.jar = function (tokens, url) {
```

Comment:
For comparison/contrast with `chromeCookies`, maybe this should be called `requestCookies`/`requestCookieJar`/`requestJar` (although I wince that `request` is an overloaded term between the `request` library and the `request` action)

Naming is hard.

Reply:
`jarForRequest` is the best I could come up with 🤷‍♂️